15_cluster/cluster_quiz_2.Rmd

```diff
@@ -0,0 +1,23 @@
+Question
+========
+  
+Wofür steht WCSS bei der Clusteranalyse?
+  
+Answerlist
+----------
+* Without Cluster Sum of Squares.
+* Within Cluster Sum of Squares.
+
```

Comment:
* Without Cosmic Sons and Sources

Nur ein Witz!

Reply:
:-)